src/citrine/informatics/design_spaces/formulation_design_space.py

```diff
@@ -48,7 +48,7 @@ class FormulationDesignSpace(Resource['FormulationDesignSpace'], DesignSpace):
         properties.String,
         properties.Set(properties.String)
     ), 'config.labels')
-    constraints = properties.Set(properties.Object(Constraint), 'config.constraints')
+    constraints = properties.List(properties.Object(Constraint), 'config.constraints')
```

Comment:
This should be a `Set`, not a `List`. I was messing around and changed it back to the wrong thing.